test/integration/appTemplates.spec.ts

```diff
@@ -33,13 +34,12 @@ test('can use images app template', async ({ page }) => {
   const breedInputLocator = page.locator('[aria-haspopup="listbox"]').first();
   await breedInputLocator.click();
   await page.locator('li:has-text("australian")').click();
+  await waitForDataResponse(page, baseURL!);
 
   const subBreedInputLocator = page.locator('[aria-haspopup="listbox"]').nth(1);
   await subBreedInputLocator.click();
-  await Promise.all([
-    page.locator('li:has-text("shepherd")').click(),
-    page.waitForResponse('https://images.dog.ceo/**'),
-  ]);
+  await page.locator('li:has-text("shepherd")').click();
+  await waitForDataResponse(page, baseURL!);
 
   const imageLocator = page.locator('img');
   expect(await imageLocator.getAttribute('src')).toContain(
```

Comment:
@apedroferreira Can you try

```suggestion
  await expect(imageLocator.getAttribute('src')).toContain(
```